webrender/src/render_backend.rs

```diff
@@ -105,6 +105,59 @@ impl Document {
     }
 }
 
+struct WebGL {
+    last_id: WebGLContextId,
+    contexts: HashMap<WebGLContextId, GLContextWrapper, BuildHasherDefault<FnvHasher>>,
```

Comment:
I just followed the existing code. Would be great to convert it in bulk to a shorter form as you propose, just in a separate PR